smartapps/samsung/sleepsense-fall-asleep.src/sleepsense-fall-asleep.groovy

```diff
@@ -0,0 +1,151 @@
+/**
+ *  SLEEPsense-fall asleep
+ *
+ *  Copyright 2015 Samsung Mobileapp
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+definition(
+    name: "SLEEPsense-fall asleep",
+    namespace: "samsung",
+    author: "Samsung Mobileapp",
+    description: "When you fall asleep, do this.",
+    category: "My Apps",
+    iconUrl: "https://s3.amazonaws.com/smartapp-icons/Convenience/Cat-Convenience.png",
+    iconX2Url: "https://s3.amazonaws.com/smartapp-icons/Convenience/Cat-Convenience@2x.png",
+    iconX3Url: "https://s3.amazonaws.com/smartapp-icons/Convenience/Cat-Convenience@2x.png")
+
+
+
+preferences {
+	page(name:"Mainpage", install:true, uninstall:true) {
+        section("When you fall asleep, do this.") {
```

Comment:
Title isn't appropriate. This is the section for selection a sleepSensor; title should reflect this.